### ISSUE TITLE
add install only support/docs

### ISSUE THREAD

**Issue by tomkralidis** [wmo-im/wis2box#230]
## User story
As a wis2box user, I want to install (but not build) wis2box directly from pre-built Docker images and minimal configuration, so that I can save time and bandwidth on building required images.

## Acceptance criteria

- [x] user is able to download the required images, set minimal config and start

## Definition of done

- [ ] unit/functional tests added/updated
- [ ] documentation updated
- [ ] live demonstration provided

**Comment by tomkralidis** [comment]
Consider adding artifact `wis2box-min-x.y.z.zip` with **just** required configurations.

**Comment by tomkralidis** [comment]
Depends on #213

**Comment by tomkralidis** [comment]
Add docs BEFORE release for user install.

**Comment by tomkralidis** [comment]
- initial setup for 0.4.0 is available for testing
- additional required containers created/published
- release management steps documented at https://github.com/wmo-im/wis2box/wiki/ReleaseManagement

Basic user workflow would be:

```bash
# download https://github.com/wmo-im/wis2box/releases/download/0.4.0/wis2box-setup-0.4.0.zip
unzip wis2box-setup-0.4.0.zip
cd wis2box-setup-0.4.0
vi dev.env (adjust WIS2BOX_HOST_DATADIR)
python wis2box-ctl.py start
```

Will continue to document in sprint 7 in the context of decoupling a "user guide" and quickstart (the above would fit into the former).

**Comment by maaikelimper** [comment]
I've tested zip-install and it mostly works. I have some comments/suggestions for ease-of-use based on my experience:
- Provide a more extensive sample dev.env, I always end up overriding a lot more than just WIS2BOX_HOST_DATADIR
- Provide data-mappings.yml.bufr data-mappings.yml.csv from the config_examples/ directory to help user to quickly setup their data-mappings.yml
- Provide example metadata surface-based-observations.yml (at least until we have a metadata-editor implemented in wis2box-admin-ui)
- set 80:80 for web-proxy ports in docker/docker-compose.override.yml

**Comment by maaikelimper** [comment]
Example of my extended dev.env :

```
# please define a data-directory on your host machine
# this will map to /data/wis2box on the wis2box-container
WIS2BOX_HOST_DATADIR=/Users/tomkralidis/Dev/wmo/wis2box/pubsub-bufr/tests/data
WIS2BOX_DATADIR_DATA_MAPPINGS=/data/wis2box/data-mappings.yml

# Optional
# Environment variable overrides
WIS2BOX_LOGGING_LOGLEVEL=WARNING
WIS2BOX_DATA_RETENTION_DAYS=30

# define the url used by the wis2box-ui and wis2box-api
WIS2BOX_URL=http://127.0.0.1
WIS2BOX_API_URL=${WIS2BOX_URL}/oapi

# PubSub local broker setup for internal commmunication
# please update to user secure username/password
WIS2BOX_BROKER_USERNAME=wis2box
WIS2BOX_BROKER_PASSWORD=wis2box

# use local broker also for public message publishing
# when using external broker, please replace this with the connection-string for your external broker
WIS2BOX_BROKER_PUBLIC=mqtt://${WIS2BOX_BROKER_USERNAME}:${WIS2BOX_BROKER_PASSWORD}@mosquitto:1883

# setup the wis2box storage username password
# please update to user secure username/password
WIS2BOX_STORAGE_USERNAME=minio
WIS2BOX_STORAGE_PASSWORD=minio123
WIS2BOX_STORAGE_DATA_RETENTION_DAYS=30

# use minio for storage
MINIO_ROOT_USER=${WIS2BOX_STORAGE_USERNAME}
MINIO_ROOT_PASSWORD=${WIS2BOX_STORAGE_PASSWORD}
MINIO_NOTIFY_MQTT_USERNAME_WIS2BOX=${WIS2BOX_BROKER_USERNAME}
MINIO_NOTIFY_MQTT_PASSWORD_WIS2BOX=${WIS2BOX_BROKER_PASSWORD}
MINIO_NOTIFY_MQTT_BROKER_WIS2BOX=tcp://${WIS2BOX_BROKER_HOST}:${WIS2BOX_BROKER_PORT}
```

**Comment by tomkralidis** [comment]
@maaikelimper I've added the sample configurations and updates as suggested, in 84206e2e9689234e367f429e26775ae72b7dc513

We can update the docs once we verify that the [0.5.0 release bundle](https://github.com/wmo-im/wis2box/releases/download/0.5.0/wis2box-setup-0.5.0.zip) is functional.

**Comment by tomkralidis** [comment]
Addressed by #349
